core/model.py

```diff
@@ -100,6 +116,9 @@ def __init__(self, path=DEFAULT_MODEL_PATH):
 
         logger.info('Loaded all models')
 
-    def predict(self, x, model):
+    def _predict(self, args):
+        model = args['model']
+        input_data = args['file']
+        x = load_array(input_data)
         logger.info('Predicting from model: {}'.format(model))
-        return self.models[model].predict(x)
+        return self.models[model]._predict(x)
```

Comment:
Are you sure we should change this line? We do not call private functions from other objects and I think `predict` should already [have `_predict` wrapped in it](https://github.com/IBM/MAX-Framework/blob/master/maxfw/model/model.py#L22)?